static/app/utils/analytics/samplingAnalyticsEvents.tsx

```diff
@@ -27,6 +27,15 @@ export type SamplingEventParameters = {
     conditions: SamplingInnerName[] | string[];
     project_id: string;
   };
+  'sampling.settings.modal.specify.client.rate_cancel': {
+    project_id: string;
+  };
+  'sampling.settings.modal.specify.client.rate_next': {
+    project_id: string;
+  };
+  'sampling.settings.modal.specify.client.rate_read_docs': {
+    project_id: string;
+  };
```

Comment:
if I apply these changes it will no longer be consistent with the other properties we have

Reply:
damn, I wonder why those keys are like that
I wish we would use dot to go deeper in the structure and underscore for space between words

`sampling.settings.modal.specify.client.rate_cancel`
vs
`sampling.settings.modal.specify_client_rate.cancel`

Reply:
yes I never really understood why we use one or another... when working on the first PR to add this structure I thought it would be best to use underscore to separate actions. something like "*_action".

Maybe we can discuss this in the frontend tsc? or do you know of some documentation we can check?